### DISCUSSION TITLE
How to change properties in GDScript for JoltHingeJoint3D?

### DISCUSSION THREAD

**Discussion by yosimba2000** [godot-jolt/godot-jolt#713] [Questions] [answered]
Is there API documentation? I've tried following the example project where JoltGeneric6DOFJoint3D properties are manipulated by code similar to `pick_joint["angular_limit_x/enabled"] = false`

I'm trying to turn on the motor on JoltHingeJoint3D instance called `door`.
`door["motor/enabled"]=true`
But it gives error `invalid set index 'motor/enabled'`

**Comment by mihe** [comment, marked as answer]
If you hover over a property in the property inspector you'll see its actual property name spelled out.

You can also just right-click any node in the scene outliner and click "Open Documentation", where you'll see all the properties and methods. You can also press F1 and type in any class name. You should also be able to Ctrl+click any class name within the editor's own script editor and it should take you to the documentation as well, although that shortcut might be different on different platforms.

I've been meaning to add some sort of separate document to actually describe the various properties and stuff, since GDExtension [doesn't support](https://github.com/godot-jolt/godot-jolt/issues/390) in-editor descriptions, but it's ended up being one of the many things sitting in my backlog at the moment.

Anyway, to answer your specific problem, it's `door["motor_enabled"]`, or just `door.motor_enabled` I guess, since that one isn't a sub-property like the axis-specific ones.

**Reply by yosimba2000** [reply]
extremely useful, thank you so much!
